pkg/processor/build/builder.go

```diff
@@ -547,6 +561,7 @@ func (b *Builder) createProcessorDockerfile() (string, error) {
 		"isDir":         common.IsDir,
 		"objectsToCopy": b.getObjectsToCopyToProcessorImage,
 		"baseImageName": func() string { return b.processorImage.baseImageName },
+		"scriptToRun":   func() string { return b.processorImage.scriptPathToRunDuringBuild },
```

Comment:
`scriptNameToRun` <-- here I would've never known this was just a file name, I thought it was a path til I saw the use

Reply:
OK